### DISCUSSION TITLE
Can't see my GitHub education community badges

### DISCUSSION THREAD

**Discussion by MadhushaPrasad** [community/community#29401] [Other Feature Feedback, Questions, & Ideas]
Hello everyone.!
How can I see my  GitHub education community badges the GitHub Education Community Forum had after moving to GitHub Discussions?
Is it possible to see it now?

**Comment by samuel-ma** [comment]
Hi @MadhushaPrasad 
This might be a recent change.
If your talking about the Pro badge you get after receiving a student's package
Then you should that it's been discontinued.......

**Reply by MadhushaPrasad** [reply]
ok thank you

**Comment by MadhushaPrasad** [comment]
ok thank you @samuel-ma

**Reply by samuel-ma** [reply]
Your welcome @MadhushaPrasad 
Hope this helped out

**Comment by IndudiniThennakoon** [comment]
Thank you for the information guys
